src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
+    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.BlazorWebAssembly" />
+  </ItemGroup>
+
+  <!-- Restore workload manifests via PackageReference -->
+  <ItemGroup>
+    <PackageReference Include="@(BundledManifests)" />
+  </ItemGroup>
+
+  <Target Name="LayoutManifests"
+        DependsOnTargets="LayoutManifestsForSDK;LayoutManifestsForMSI" />
+
+  <Target Name="LayoutManifestsForSDK"
+          DependsOnTargets="SetupBundledComponents;GenerateManifestVersions">
+    
+    <Copy SourceFiles="@(ManifestContent)"
+          DestinationFolder="$(RedistLayoutPath)sdk-manifests/$(CliProductBandVersion)00/%(DestinationPath)"/>
+  
+  </Target>
+
+  <Target Name="LayoutManifestsForMSI"
```

Comment:
we are creating manifest MSI inside SDK?

We should do that for Blazor since it happens to be part of SDK repo. But it won't scale well to package everyone's manifest to MSI

Reply:
Yes, for now. This creates a MSI that includes the blazor and xamarin manifests but once Jacques's MSI generation tool is finished we won't need this anymore.